### DISCUSSION TITLE
DONE - Add dialogue prompt asking user to leave rating / review in App Store

### DISCUSSION THREAD

**Discussion by pretyflaco** [GaloyMoney/galoy#1362] [Bounties]
## Description
Implement an In-App dialogue for galoy-mobile that prompts users to leave a public rating + review of the app in respective App Store.

### Possible flow:
Asking the user if they are enjoying the app.
If yes:
2. Ask if they want to review.
3. If yes, prompt for a review.

If no:
2. Ask if they need help from support.
3. If yes, bring them to support.

## Background
Apps can be rated and reviewed by users of the app for other prospective users to read in the Google Play, Apple and Huawei App Stores. Apps with high ratings are preferred by users, however rating and reviewing an app is time consuming and users need to be reminded to do it. In order to get a better rating, the timing of the reminder prompt could play a role and should be considered. Read more about this [here](https://www.reactnativeschool.com/best-practices-for-capturing-user-feedback-in-react-native). 

Potentially helpful: Library which ties into the native android/ios rating systems called [react-native-rate](https://www.npmjs.com/package/react-native-rate)

## Task
Submit a PR that implements the above described feature into galoy-mobile. 

## Definition of Done
Submit PR that leads to closure of [this issue](https://github.com/GaloyMoney/galoy-mobile/issues/6)

## Bounty
3M Sats (0.03 BTC)

## Bounty Patrons
[Galoy](https://galoy.io/)

**Comment by serigneabdou** [comment]
I am interested
